application/helpers/surveytranslator_helper.php

```diff
@@ -221,13 +232,15 @@ function getLanguageData($bOrderByNative = false, $sLanguageCode = 'en')
     $supportedLanguages['prs']['dateformat'] = 6;
     $supportedLanguages['prs']['radixpoint'] = 0;
     $supportedLanguages['prs']['cldr'] = 'fa_af';
+    $supportedLanguages['prs']['momentjs'] = null;
```

Comment:
null?

Reply:
Arg resolve don't comaent at same time.

Yes null/not set/empty mean take `en`, language not in https://momentjs.com/ .

https://github.com/LimeSurvey/LimeSurvey/pull/1420/files#diff-fd7e1638b849ebd4dfc8692ff3e535c7R1158 

Can take other, buty since i don't speak this one …

See manual : https://manual.limesurvey.org/Translating_LimeSurvey#Other_part_to_be_translated_2